nuxeo-management-rest-api/src/main/java/org/nuxeo/rest/management/elasticsearch/ElasticsearchObject.java

```diff
@@ -99,6 +100,44 @@ public Response getIndexingStatus(@PathParam("indexingId") String indexingId) {
         return Response.status(SC_OK).entity(indexingStatus).build();
     }
 
+    /**
+     * Executes an ES flush on document index of a given repository.
+     */
+    @POST
+    @Path("{repositoryName}/flush")
+    public void doFlush(@PathParam("repositoryName") String repositoryName) {
```

Comment:
I guess we also want to have the repo name as a `QueryParam`.

Reply:
you right 👍